tests/unit/display.t

```diff
@@ -150,4 +151,59 @@ $product_ref = {
 $expected = lang('to_do_status') . separator_before_colon($lc) . q{:};
 like(display_field($product_ref, 'states'), qr/$expected/);
 
+# should not loose the second facet at the end of the url on redirection
+my $facets_ref = {
+	'current_link' => 'category/en:lemonades/data-quality',
```

Comment:
current_link will be set by display_tag(), so the value you set here will be overwritten, we can remove it.

we should add a tagtype => "categories" though, otherwise you get a //lemonade/data-quality link, instead of /category/lemonade/data-quality

Can you use "bread" instead of "lemonades"? There's an issue in the categories taxonomy, the canonical name should be in plural (lemonades), but currently it's in singular.